doc/index.rst

```diff
@@ -1,25 +1,52 @@
-Welcome to yaml2sbml's documentation!
-===============================================
-
-This package allows the user to convert a system of ODEs specified in a yaml file to SBML. In addition, if experimental data is provided in the yaml file it can also be converted to PEtab.
-
-
-Installation
----------------------------
-
-`yaml2sbml` can be installed with `pip`:
-
-```pip install yaml2sbml```
-
+yaml2sbml
+=========
+
+.. image:: https://github.com/yaml2sbml-dev/yaml2sbml/workflows/CI/badge.svg
+   :target: https://github.com/yaml2sbml-dev/yaml2sbml/actions
+   :alt: Build status
+.. image:: https://codecov.io/gh/yaml2sbml-dev/yaml2sbml/branch/master/graph/badge.svg
+   :target: https://codecov.io/gh/yaml2sbml-dev/yaml2sbml
+   :alt: Code coverage
+.. image:: https://app.codacy.com/project/badge/Grade/dc25c9a84ba54710bbb23a6e08ab5d22
+   :target: https://www.codacy.com/manual/martamatos/yaml2sbml/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=yaml2sbml-dev/yaml2sbml&amp;utm_campaign=Badge_Grade
+   :alt: Code quality
+
+:Release: |version|
+:Source code: https://github.com/yaml2sbml-dev/yaml2sbml
+
+.. image:: logo/logo_yaml2sbml_long.png
+   :alt: yaml2sbml logo
+   :align: center
+   :scale: 30 %
+
+**yaml2sbml** allows the user to convert a system of ODEs specified in a yaml
```

Comment:
```suggestion
**yaml2sbml** allows the user to convert a system of ODEs specified in a YAML
```

Reply:
especially, since we also use capital letters for SBML

Reply:
maybe also add a link for SBML & PEtab

Reply:
added links for yaml, sbml, petab